docs/dev-guides/3-distributing-xrpackage.md

```diff
@@ -3,44 +3,75 @@ id: distributing-xrpackage
+
+### What is Ethereum?
+
+<a href="https://ethereum.org/" target="_blank" rel="noopener noreferrer">Ethereum</a> is a decentralized open-source platform for writing and distributing blockchain applications, known as _decentralized apps (dapps)_. Dapps are backed by Ethereum smart contracts, which are transaction protocols that enforce a contract, and rely on the blockchain to run a trustless, autonomous peer-to-peer network that stores an immutable record of all transactions. Data is stored as a ledger of _blocks_ that are cryptographically linked together - each block contains a cryptographic hash to the previous block and a timestamp. This means the data cannot be altered once it has been published. In addition, the most recent state of each smart contract is stored along with a complete record of ether transactions.
+
+The Ethereum network has a native cryptocurrency ('digital money') called <a href="https://ethereum.org/eth/" target="_blank" rel="noopener noreferrer">Ether (ETH)</a>. Ether is used as the fuel to cover cost of fees in exchange for usage of network processing power. This ether fee, calculated in units of Ethereum gas, will depend on the amount of computational power and time required for the action being processed. Ether is validated by miners who provide 'proof-of-work' for the Ethereum network and create new Ether tokens by validating smart contracts. The faster the hashrate of the miners, the more secure the Ethereum system. Like other cryptocurrencies, Ethereum can be used to buy and sell Ether tokens via blockchain technology.
+
+Ethereum gives ownership and interoperability of your packages &mdash; you will always have a complete sense of ownership for anything you create and can provably sell anything you own. As a reliable decentralized platform, Ethereum serves to provide complete freedom with blockchain technology to build and distribute applications without third-party interferences.
+
+#### Using the Webaverse Blockchain
+
+**Note:** The publish functionality in the XRPackage CLI currently only publishes to the [Rinkeby](https://www.rinkeby.io/) testnet. You will need to have a sufficient ETH balance in order to publish packages. You can get free Rinkeby testnet ETH <a href="https://faucet.rinkeby.io/" target="_blank" rel="noopener noreferrer">at the faucet</a>.
+
+The following steps assume you have at least a basic familiarity with the Ethereum network; we will login to our Ethereum wallet and publish our packages to the network.
+
+First, use the `login` command to either login to an existing wallet, or create a new wallet. This is the wallet we will be using to pay the gas fees for publishing our package. If you create a new wallet using the CLI, make sure to visit the faucet and load it with ETH before attempting to publish a package.
```

Comment:
I believe the command is actually `xrpk wallet` -- login is only for the web registry, not the blockchain.